### DISCUSSION TITLE
Eu shops not working when filtering for series

### DISCUSSION THREAD

**Discussion by kiurious** [jef/streetmerchant#3167] [Help]
Hello,
I have added all EU shops found at: https://www.jef.buzz/streetmerchant/reference/filter/ to the filter and selected 4090 as series and it gives me the message that `"some of the selected stores (grayed out) dont have what you are looking for"` to all of the shops. If I add, US-based amazon,newegg etc, then it starts looking for those. I went in to currys and some other websites in EU and they had 4090's in stock. I tried with a 4080 and the result was the same. 

Please see screenshot. Any help will be appreciated.
![Screenshot 2023-08-21 145115](https://github.com/jef/streetmerchant/assets/29750764/d3b905b2-26b5-483a-b085-6605dc278a1c)

Thank you

**Comment by Mr-Tech-13** [comment]
@kiurious Those stores dont have 4090 links. You can add them though. Thats what the "some of the selected stores (grayed out) dont have what you are looking for" means
